### PR TITLE
Missing annotations added on responses module.

### PR DESCRIPTION
**Pull request by marcosschroh**
command to run: `mypy starlette/responses.py --disallow-untyped-defs --ignore-missing-imports --follow-imports=silent`

### CODE REVIEW

File: starlette/responses.py [hunk 1]
```diff
@@ -19,9 +19,9 @@
 try:
     import ujson as json
 
-    JSON_DUMPS_OPTIONS = {"ensure_ascii": False}
+    JSON_DUMPS_OPTIONS: typing.Dict[str, typing.Any] = {"ensure_ascii": False}
 except ImportError:  # pragma: nocover
-    import json
+    import json  # type: ignore
```
Comment (lovelydinosaur) [inline starlette/responses.py]: If mypy is forcing us to add this kind of noise then I'm :-/

File: starlette/responses.py [hunk 2]
```diff
@@ -19,9 +19,9 @@
 try:
     import ujson as json
 
-    JSON_DUMPS_OPTIONS = {"ensure_ascii": False}
+    JSON_DUMPS_OPTIONS: typing.Dict[str, typing.Any] = {"ensure_ascii": False}
```
Comment (lovelydinosaur) [inline starlette/responses.py]: Can we avoid this?

File: starlette/responses.py (hunk 1 above)
Comment (marcosschroh) [reply starlette/responses.py]: Yes, me too. Seems that is the only solution. You can take a look [here](https://github.com/python/mypy/issues/1153)

File: starlette/responses.py (hunk 2 above)
Comment (marcosschroh) [reply starlette/responses.py]: We need to define the type. I can do this if you want:

```python
JSON_DUMPS_OPTIONS = {"ensure_ascii": False}  # type: typing.Dict[str, typing.Any]
```

**Comment by lovelydinosaur** [comment]
Good stuff. This'd need updating to the latest master particularly since #99 made some changes here.

**Comment by marcosschroh** [comment]
Hi,

Now is updated with `master`!!

**Pull request merged**